docs/_guide/publish.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: guide
+title: Publish an element
+slug: publish
+---
+
+{::options toc_levels="1..3" /}
+* ToC
+{:toc}
+
+This page describes how to publish a LitElement component to npm.
+
+We recommend publishing JavaScript modules in standard ES2017. If you're writing your element in standard ES2017, you don't need to transpile for publication. If you're using TypeScript, or ES2017+ features such as decorators or class fields, you will need to transpile your element for publication.
+
+## Publishing to npm
+
+To publish your component to npm, [see the instructions on contributing npm packages](https://docs.npmjs.com/packages-and-modules/contributing-packages-to-the-registry).
+
+Your package.json configuration should have both the `main` and `module` fields:
+
+**package.json**
+
+```json
+{
+  "main": "my-element.js",
+  "module": "my-element.js"
+}
+```
+
+You should also create a README describing how to consume your component. A basic guide to consuming LitElement components is documented at [Use a component](use).
+
+## Transpiling with TypeScript
+
+When compiling your code from TypeScript to JavaScript, we recommend targeting ES2017 with Node.js module resolution. See the examples below for suggested options in tsconfig.json, and see also the [tsconfig.json documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) for more information.
+
+### Targeting ES2017 (Recommended)
+
+The following JSON sample is a partial tsconfig.json that uses recommended options for targeting ES2017:
+
+```json
+  "compilerOptions": {
+    "target": "ES2017",
+    "module": "ES2017",
+    "moduleResolution": "node",
+    "lib": ["ES2017", "DOM"],
+    "experimentalDecorators": true,
+    "outDir": "path/to/your-output-dir"
+  }
+```
+
+### Targeting ES5
+
+The following JSON sample is a partial tsconfig.json for those who need to target ES5:
```

Comment:
Might be worth repeating that we recommend letting applications compile dependencies to ES5 with Babel. This could lead some developers to compiling to ES5 before publishing to npm.

Also, we should probably mention that compiling to ES5 means that you _have_ to include the custom-elements-es5-adapter on browsers that have native Custom Elements support.

Reply:
Took it out bc it's not to do with publishing